src/components/v2/Tabs/index.tsx

```diff
@@ -54,7 +54,7 @@ export const Tabs = ({
             onClick={() => handleChange(index)}
             css={styles.getButton({
               active: index === activeTabIndex,
-              last: index === title.length - 1,
+              last: index === tabsContent.length - 1,
```

Comment:
@therealemjy here is the fix from your PR

Reply:
Thanks